doc/tutorials/introduction/linux_install/linux_install.markdown

```diff
@@ -4,7 +4,7 @@ Installation in Linux {#tutorial_linux_install}
 @next_tutorial{tutorial_linux_gcc_cmake}
 
 
-The following steps have been tested for Ubuntu 10.04 but should work with other distros as well.
+The following steps have been tested for Ubuntu 10.04, 16.04 LST, 18.04 LST, CentOS 8, Redhat 8  and openSUSE Leap 15.2 but should work with other distros as well.
```

Comment:
I have found only `Red Hat Enterprise Linux 8`. There is no just `Redhat 8`

Reply:
Done, thanks.